.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/common"]
```

Comment:
are there alternatives to git submodules?

Reply:
Having it as a normal dependency means we need to do all the webpack configuration there too and then build the distribution files and then publish to npm. Which I thought is a bad thing because of maintaining 2 different configuration files. This is definitely better.

Reply:
OK, as long as the CI/build handles these, I'm ok with this :)